modules/ocf_tv/files/X11/xorg.conf

```diff
@@ -1,5 +1,7 @@
 Section "Device"
    Identifier  "Intel Graphics"
    Driver      "intel"
+   Option      "AccelMethod"    "sna"
    Option      "TearFree"    "true"
+   Option      "DRI"    "3"
```

Comment:
I don't think this will work on jessie (see "reverting to software rendering" in xorg log) and shouldn't be needed in jessie-backports and stretch, where it sets the maximum DRI level I think (default is no maximum).